.gitignore

```diff
@@ -1,8 +1,10 @@
+# generated
+*.tx
```

Comment:
`*.tx` files haven't been a thing since like 2013 or 2014.. Where did this come from?

Reply:
Copied from pokecrystal's .gitignore. If that's outdated, I'll remove it from the red+gold+crystal repos.